internal/client/client.go

```diff
@@ -414,7 +414,7 @@ func (c *CSAPI) LoginUser(t *testing.T, localpart, password string) (userID, acc
 			"user": localpart,
 		},
 		"password": password,
-		"type": "m.login.password",
+		"type":     "m.login.password",
```

Comment:
My editor insisted on this change, I can edit the file here if it's a problem.

Reply:
Can we revert this change please?